### PR TITLE
CON-4800 Increase Unix socket timeout for multipathd communication fr…

### PR DESCRIPTION
**Pull request by diganta-001**
…om node/init container, one additional task will be to  add env variable for helm

### CODE REVIEW

File: tunelinux/config/multipath.conf.generic [hunk 1]
```diff
@@ -1,4 +1,5 @@
 defaults {
+    uxsock_timeout      300000
```
Comment (datamattsson) [inline tunelinux/config/multipath.conf.generic]: Can we make this change in multipath.conf.upstream too and make sure the keywords are in the same place etc.

**Review by datamattsson — CHANGES_REQUESTED** [review]

**Review by kalikivayi — CHANGES_REQUESTED** [review]
I have reviewed manually as well as with the AI MCP. I do not have any manual comments, but AI has given few comments. Please address them.

[con-4800_review.md](https://github.com/user-attachments/files/29883756/con-4800_review.md)

File: tunelinux/config/multipath.conf.generic (hunk 1 above)
Comment (diganta-001) [reply tunelinux/config/multipath.conf.generic]: file added

Comment (diganta-001) [reply tunelinux/config/multipath.conf.generic]: > Can we make this change in multipath.conf.upstream too and make sure the keywords are in the same place etc

file added , thanks

**Comment by diganta-001** [comment]
> I have reviewed manually as well as with the AI MCP. I do not have any manual comments, but AI has given few comments. Please address them.
> 
> [con-4800_review.md](https://github.com/user-attachments/files/29883756/con-4800_review.md)

Here is the Copilot and developer response 
1 (defaults section) / 4 (no reload) — Not valid concerns. setMultipathRecommendations always creates the defaults{} section if missing and always reaches the uxsock_timeout reconciliation block. SetMultipathRecommendations unconditionally calls multipathd reconfigure after saving, so the value takes effect immediately on a running daemon.
2 (justify 300s) — Will add a doc comment; the 5-minute value will also be captured in the docs, so no action needed in code itself.
3  (upper bound / clamp) — Not adding a hard max. Capping it in code is actually risky — a hardcoded ceiling could itself become the wrong value for some environment/scale, and would require another code change to adjust. The env var (MULTIPATH_UXSOCK_TIMEOUT) already gives admins full control to tune it themselves if 300000ms isn't right for their setup, so this is intentionally left open rather than artificially bounded.
5 (const/template duplication) + missing tests — tunelinux/ has no existing test coverage for any function today (verified — zero _test.go files, no git history), so adding new tests here isn't restoring parity with an existing convention. Treating as a follow-up, not a blocker for this PR.

**Review by kalikivayi — APPROVED** [review]
LGTM

**Review by rohit-balakrishna — APPROVED** [review]
@diganta-001 , please share the e2e test results with this patch and the GetSnapshots change. Thanks

**Comment by rohit-balakrishna** [comment]
> @diganta-001 , please share the e2e test results with this patch and the GetSnapshots change. Thanks

CSI E2E after applying both changes in PR226 and PR229 to the CSI Driver and 3par CSP passes and results are:
`------------------------------

Ran 99 of 7907 Specs in 6289.825 seconds
SUCCESS! -- 99 Passed | 0 Failed | 0 Pending | 7808 Skipped
PASS
Created /data/csi-e2e-vpl6f-26-07-14-06-43-03.xml.gz
rohitb@rb-esx-master:~/CSI-E2E/csi-e2e$
`
@datamattsson please approve.

**Review by datamattsson — APPROVED** [review]
